features/payment/managing_payments/accessing_payments_order_from_payment_index.feature

```diff
@@ -0,0 +1,19 @@
+@managing_payments
+Feature: Accessing payment's order from the payment index
+    In order to make payment and orders management more fluent
+    As an Administrator
+    I want to be able to access order's page from payments index
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "UPS" shipping method with "$10.00" fee
```

Comment:
Here and in other your features:
```suggestion
        And the store ships everywhere for free
```
because the information about shipping method name and fee is irrelevant